pkg/porter/archive.go

```diff
@@ -186,6 +183,27 @@ func (ex *exporter) addImage(image bundle.BaseImage) error {
 	return checkDigest(image, dig)
 }
 
+// createArchiveFolder set up a temporary directory for storing all data needed to archive a bundle.
+// It sanitizes the name and make sure only the current user has full permission to it.
+// If the name contains a path separator, all transit paths are created.
+func (ex *exporter) createArchiveFolder(name string) (string, error) {
+	cleanedPath := afero.UnicodeSanitize(name)
+	tmpDir := filepath.Join(os.TempDir(), filepath.Dir(cleanedPath))
```

Comment:
The reason why I couldn't directly use the `TmpDir` is because it doesn't create nested folders. I changed the function to replace all path separators to a dash. Let me know what do u think

Reply:
That sounds like a good way to deal with it, thanks! 👍